src/test/regress/sql/executor_local_failure.sql

```diff
@@ -0,0 +1,30 @@
+CREATE SCHEMA failure_local_modification;
```

Comment:
https://app.circleci.com/pipelines/github/citusdata/citus/33947/workflows/64f52f73-dbef-45c7-8df5-58315bea5106/jobs/1194902 << flaky test detector is failing because we are not cleaning up the schema used for the test.

I would suggest drop cascading the whole schema at the end to fix the flaky test detector